### PR TITLE
Add filter instead of gif

### PR DESCRIPTION
**Pull request by Adeodonne**
## Description
Add filter instead of gif on streetcode page
image

dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: src/features/AdminPage/NewStreetcode/MainBlock/FileInputsPart.component.tsx [hunk 1]
```diff
@@ -267,7 +270,7 @@ const FileInputsPart = ({ form, onChange }: FileInputsPartProps) => {
                                     || name.endsWith('.jpg') || name === '') {
                                     return Promise.resolve();
                                 }
-
+                                // eslint-disable-next-line max-len
                                 return Promise.reject(Error('Тільки файли з розширенням webp, jpeg, png, jpg дозволені!'));
```
Comment (KateYatsiuk) [inline src/features/AdminPage/NewStreetcode/MainBlock/FileInputsPart.component.tsx:274]: If you move 'Тільки файли з розширенням webp, jpeg, png, jpg дозволені!' to constant, you can avoid adding // eslint-disable-next-line max-len. Moreover, this message is used twice in this component.
**Try to solve eslint warnings (if possible), not just ignore them.**

**Review by KateYatsiuk — APPROVED** [review]
Well done

File: src/features/AdminPage/NewStreetcode/MainBlock/FileInputsPart.component.tsx [hunk 2]
```diff
@@ -209,25 +209,28 @@ const FileInputsPart = ({ form, onChange }: FileInputsPartProps) => {
                                     } else if (file.name) {
                                         name = file.name.toLowerCase();
                                     }
-                                    if (name.endsWith('.gif') || name.endsWith('.webp') || name === '') {
+                                    if (name.endsWith('.jpeg') || name.endsWith('.png') || name.endsWith('.webp')
```
Comment (Oleh-Kril) [inline src/features/AdminPage/NewStreetcode/MainBlock/FileInputsPart.component.tsx:212]: I would suggest to create something like this: type image = name.endsWith('.jpeg') || name.endsWith('.png') || name.endsWith('.webp') || name.endsWith('.jpg') || name === '' and use it here and other components

File: src/features/AdminPage/NewStreetcode/MainBlock/FileInputsPart.component.tsx [hunk 3]
```diff
@@ -209,25 +209,28 @@ const FileInputsPart = ({ form, onChange }: FileInputsPartProps) => {
...
-                        accept=".gif,.webp"
+                        accept=".jpeg,.png,.jpg,.webp"
                         listType="picture-card"
                         multiple={false}
                         maxCount={1}
                         fileList={animation}
                         beforeUpload={(file) => {
-                            const isGif = file.type === 'image/gif' || file.type === 'image/webp';
-                            if (!isGif) {
+                            const isValid = (file.type === 'image/jpeg') || (file.type === 'image/webp')
```
Comment (Oleh-Kril) [inline src/features/AdminPage/NewStreetcode/MainBlock/FileInputsPart.component.tsx:231]: please extract it to type and use it on line 346

**Review by Oleh-Kril — APPROVED** [review]
Please remind me to discuss deleting all gifs in databases

**Comment by Adeodonne** [comment]
DO NOT MERGE

**Review by ValDekh — APPROVED** [review]

**Pull request merged**